modules/aruco/test/test_charucodetection.cpp

```diff
@@ -745,15 +745,16 @@ TEST(Charuco, issue_14014)
     ASSERT_EQ(corners.size(), 19ull);
     EXPECT_EQ(Size(4, 1), corners[0].size()); // check dimension of detected corners
 
-    ASSERT_EQ(rejectedPoints.size(), 21ull);
+    size_t numRejPoints = rejectedPoints.size();
+    ASSERT_TRUE(numRejPoints > 0ull);
```

Comment:
I recommend to stay with the original assertion, but change value.

Reply:
I think, that the exact number of rejected candidates should not be verified.

Reply:
Strong checks allow to highlight regressions. You can change particular constants each time when it's needed.

Reply:
check was added